app/main.py

```diff
@@ -1,5 +1,81 @@
+from typing import List
+
+
 class Cargo:
-    def __init__(self, weight: int) -> None:
+    def __init__(
+            self,
+            weight: int
+    ) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot(Cargo):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        super().__init__(weight)
+        self.name = name
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
You could omit the check `is not None` thus simplifying your code
```suggestion
        self.coords = coords if coords else [0, 0]
```
Or use the `or expression` for the same purpose
```suggestion
        self.coords = coords or [0, 0]
```